opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Dict, Union
```

Comment:
minor: other files do keep a blank line after the copyright comment, it'll be great to keep consistent.

Reply:
thanks for pointing that out! will do.

A cursory glance doesn't show this capability, but I wonder if we can lint for this.

Reply:
```suggestion

```

This line should fail lint since these are unused, but we weren't linting the SDK by default. I add the check in https://github.com/open-telemetry/opentelemetry-python/pull/65, which means this PR will fail the check once that PR is merged.

Reply:
...and if we start writing out own pylint checkers maybe that's a signal that we've got too far and should just start using a formatter.